### PR TITLE
Fixed wording to request feedback

### PR DESCRIPTION
**Pull request by albertoac**
Suggested by Flo via Slack

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -10,8 +10,10 @@ Development
 ### Bug fixes / enhancements
 - Reassign ownership after destroying an OAuth API key ([#15162](https://github.com/CartoDB/cartodb/pull/15162))
 - Fix message in password confirmation modal when changing the password ([CartoDB/support#2187](https://github.com/CartoDB/support/issues/2187))
-- Fix message in password protected maps ([CartoDB/design#1758](https://github.com/CartoDB/design/issues/1758)
-- Fix Visualization Searcher ([CartoDB/cartodb#15224](https://github.com/CartoDB/cartodb/issues/15224)
+- Fix message in password protected maps ([CartoDB/design#1758](https://github.com/CartoDB/design/issues/1758))
+- Fix Visualization Searcher ([CartoDB/cartodb#15224](https://github.com/CartoDB/cartodb/issues/15224))
+- Fix Visualization Searcher ([CartoDB/cartodb#15224](https://github.com/CartoDB/cartodb/issues/15224))
```
Comment (alonsogarciapablo) [inline NEWS.md]: Careful. This is duplicated.

**Comment by alonsogarciapablo** [comment]
👍

File: NEWS.md (hunk 1 above)
Comment (albertoac) [reply NEWS.md]: 🤦‍♂ good catch, thank you!

File: NEWS.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ Development
 ### Bug fixes / enhancements
 - Reassign ownership after destroying an OAuth API key ([#15162](https://github.com/CartoDB/cartodb/pull/15162))
 - Fix message in password confirmation modal when changing the password ([CartoDB/support#2187](https://github.com/CartoDB/support/issues/2187))
-- Fix message in password protected maps ([CartoDB/design#1758](https://github.com/CartoDB/design/issues/1758)
-- Fix Visualization Searcher ([CartoDB/cartodb#15224](https://github.com/CartoDB/cartodb/issues/15224)
+- Fix message in password protected maps ([CartoDB/design#1758](https://github.com/CartoDB/design/issues/1758))
+- Fix Visualization Searcher ([CartoDB/cartodb#15224](https://github.com/CartoDB/cartodb/issues/15224))
+- Fix wording for feedback (Suggested by Flo via Slack)
```
Comment (elenatorro) [inline NEWS.md]: I wouldn't leave a comment here if there's no issue about it, I'll just say `Fix wording for feedback`

Comment (albertoac) [reply NEWS.md]: Thanks! Fixed

**Pull request merged**